packages/muray/muray.cpp

```diff
@@ -57,14 +125,20 @@ void MuUpdateInput(const v8::FunctionCallbackInfo<v8::Value> &args) {
         if (IsMouseButtonReleased(button)) mu_input_mouseup  (&ctx, x, y, 1 << button);
     }
 
-    char c;
-    char input[2];
-    if ((c = GetCharPressed()))
-    {
-        input[0] = c;
-        input[1] = '\0';
-        mu_input_text(&ctx, input);
+    int key;
+    while ((key = GetCharPressed())) {
+        mu_input_text(&ctx, (char[2]){(char)key, 0});
     }
+
+    if(IsKeyPressed(KEY_BACKSPACE))  mu_input_keydown(&ctx, MU_KEY_BACKSPACE);
```

Comment:
This allows to hold backspace to delete characters

```suggestion
    if(IsKeyPressed(KEY_BACKSPACE) || IsKeyPressedRepeat(KEY_BACKSPACE))  mu_input_keydown(&ctx, MU_KEY_BACKSPACE);
```